front-end/yarn.lock

```diff
@@ -2599,6 +2599,21 @@
     telejson "^3.2.0"
     util-deprecate "^1.0.2"
 
+"@substrate/context@^0.5.0":
```

Comment:
Did you ran deduplicate screen again

Reply:
nope, just a deletion of `yarn.lock` and yarn again.